app/controllers/profile_controller.rb

```diff
@@ -9,6 +13,35 @@ def show
     @user = current_user
   end
 
+  def password_form
+    @password_form = ChangePasswordForm.new({})
+  end
+
+  def change_password
+    @password_form = ChangePasswordForm.new(params[:change_password_form])
+    if @password_form.valid?
+      backend_change_password(
+        old: @password_form.old_password,
+        proposed: @password_form.password,
```

Comment:
I meant my previous comment for the `old` and `proposed` keywords to be hidden from the controller here...the controller doesn't need to be exposed to the Cognito's terminology..

Reply:
Sorry I missed this while doing all the model stuff... This has now been changed.